src/datastructures.apib

```diff
@@ -140,3 +140,121 @@
         + timeTracking.deleted
         + timeTracking.updated
         + user.deactivated
+
+## WorkOrderData (object)
```

Comment:
PP: I actually don't like that complete responses are modelled as datastructures
there is not a lot of reusability in that and it makes you have to check two files all the time
I used datastructures for things that are value-object like things